articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -449,23 +449,24 @@ From the Azure portal, follow these steps:
-    | At a specific time | Optional. Fill out to import key-values from a specific point in time. This is the point in time of the resource you selected in the previous step. Format: "YYYY-MM-DDThh:mm:ssZ".                                                    | *07/28/2022 12:00:00 AM*  |
-    | Content type       | Optional. Check the box **Override default key-value content types** and select **Key Vault Reference** or **JSON** under **Content type** to state that the imported content consists of a Key Vault reference or a JSON file. | *JSON (application/json)* |
-
-    > [!IMPORTANT]
-    > If the keys you want to export have labels, you must use the command `--label` and enter the corresponding labels. If you don't select a label, only keys without labels will be exported. Use a comma sign (`,`) to select several labels or use `*` to include all labels, including the null label (no label).
+    | Prefix             | Optional. This prefix will be trimmed from each exported key-value's "key" property. A key prefix is the beginning part of a key. Prefixes can be used to manage groups of key-values in a configuration store. Prefix will be ignored for feature flags.           | *TestApp:*                 |
+    | Export as reference | Optional. Check to export key-values to App Service as App Configuration references. [Learn more](../app-service/app-service-configuration-references.md) |
+    | At a specific time | Optional. Fill out to export key-values from a specific point in time. This is the point in time of the resource you selected in the previous step and the version of key-values that existed in the resource at the specified time will be exported. Format: "YYYY-MM-DDThh:mm:ssZ". This field defaults to the current point in time of the resource when left empty.                                                  | *07/28/2022 12:00:00 AM*  |
+    | From label         | Optional. Select an existing label to restrict your export to key-values with a specific label. If you don't select a label, only key-values with the "No label" label will be exported.                                    | *prod*                    |
 
 1. Select **Apply** to proceed with the export.
 
-You've exported key-values that have the "prod" label from an App Service resource, at their state from 07/28/2021 12:00:00 AM, and have trimmed the prefix "TestApp". The keys have been exported with a content type in JSON format.
+You've exported key-values that have the "prod" label from an App Service resource, at their state from 07/28/2021 12:00:00 AM, and have trimmed the prefix "TestApp". The key-values have been exported with a content type in JSON format.
+
+If you checked the box to export key-values as app configuration references, the exported key-values will be indicated as App Configuration references in the "Source" column of your App Service resource configuration settings.
```

Comment:
If you checked the box to export key-values as ~app configuration~ references, ...